src/Tribe/RSVP/Service_Provider.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Tribe\Tickets\RSVP;
+
+/**
+ * Class Service_Provider
+ *
+ * @since TBD
+ *
+ * @package Tribe\Tickets\RSVP
+ */
+class Service_Provider extends \tad_DI52_ServiceProvider {
+
+	/**
+	 * @inheritDoc
+	 *
+	 * @since TBD
+	 */
+	public function register() {
+		$this->register_early_access();
+	}
+
+	/**
+	 * Registers the RSVP Early Access Service.
+	 *
+	 * @since TBD
+	 */
+	private function register_early_access() {
+		add_action( 'init', [
+			$this->container->make( Early_Access::class ),
```

Comment:
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: There must be a single space before the last parenthesis
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: ddce73dcb8fb40becbc4f37a15d03064</sub>